1.16.5/src/main/java/io/github/axolotlclient/mixin/PlayerListHudMixin.java

```diff
@@ -150,4 +165,156 @@ public abstract class PlayerListHudMixin {
 	private boolean axolotlclient$alwaysShowHeadLayer(PlayerEntity instance, PlayerModelPart modelPart) {
 		return Tablist.getInstance().alwaysShowHeadLayer.get() || instance.isPartVisible(modelPart);
 	}
+
+	@Inject(
+		method = "render",
+		at = @At(
+			value = "INVOKE",
+			target = "Lnet/minecraft/client/gui/hud/PlayerListHud;renderLatencyIcon(Lnet/minecraft/client/util/math/MatrixStack;IIILnet/minecraft/client/network/PlayerListEntry;)V"
+		),
+		locals = LocalCapture.CAPTURE_FAILHARD
+	)
+	public void axolotlclient$renderWithoutObjective(
+		MatrixStack matrixStack, int argY, Scoreboard scoreboard, ScoreboardObjective scoreboardObjective, CallbackInfo ci,
+		ClientPlayNetworkHandler clientPlayNetworkHandler, List list, int i, int j, int l, int m, int k,
+		boolean bl, int n, int o, int p, int q, int r, List list2, int t, int u, int s, int v, int y, int z, PlayerListEntry playerListEntry2
+	) {
+		if (!BedwarsMod.getInstance().isEnabled() || !BedwarsMod.getInstance().isWaiting()) {
+			return;
+		}
+		int startX = v + i + 1;
+		int endX = startX + n;
+		String render;
+		try {
+			if (playerListEntry2.getProfile().getName().contains(Formatting.OBFUSCATED.toString())) {
+				return;
+			}
+
+			render = String.valueOf(HypixelAbstractionLayer.getPlayerLevel(playerListEntry2
+					.getProfile().getId().toString().replace("-", ""),
+				LevelHeadMode.BEDWARS.toString()));
+		} catch (Exception e) {
+			return;
+		}
+		this.client.textRenderer.drawWithShadow(matrixStack,
+			render,
+			(float) (endX - this.client.textRenderer.getWidth(render)) + 20,
+			(float) y,
+			-1
+		);
+	}
+
+	@Inject(
+		method = "renderScoreboardObjective",
+		at = @At(
+			value = "INVOKE",
+			target = "Lnet/minecraft/client/font/TextRenderer;drawWithShadow(Lnet/minecraft/client/util/math/MatrixStack;Ljava/lang/String;FFI)I",
+			ordinal = 1
+		),
+		cancellable = true
+	)
+	private void axolotlclient$renderCustomScoreboardObjective(
+		ScoreboardObjective objective, int y, String player, int startX, int endX,
+		PlayerListEntry playerEntry, MatrixStack matrices, CallbackInfo ci
+	) {
+		if (!BedwarsMod.getInstance().isEnabled()) {
+			return;
+		}
+
+		BedwarsGame game = BedwarsMod.getInstance().getGame().orElse(null);
+		if (game == null) {
+			return;
+		}
+		BedwarsPlayer bedwarsPlayer = game.getPlayer(playerEntry.getProfile().getName()).orElse(null);
+		if (bedwarsPlayer == null) {
+			return;
+		}
+		ci.cancel();
+		String render;
+		int color;
+		if (!bedwarsPlayer.isAlive()) {
```

Comment:
Perhaps move this into BedwarsGame so it's easier to change in the future?

Reply:
ok!